src/apps/vpn/telemetry.cpp

```diff
@@ -180,6 +181,15 @@ void Telemetry::initialize() {
         GleanSample::serverUnavailableError);
   });
 
+  connect(SettingsHolder::instance(), &SettingsHolder::startAtBootChanged, this,
+          []() {
+#if !defined(MZ_IOS)
```

Comment:
Start on boot is not a setting on iOS. This if-stmt can be removed

Reply:
Done - this was when I was trying to figure out why it failed, it's been removed